tests/manage/pv_services/test_no_space_left.py

```diff
@@ -0,0 +1,155 @@
+"""
+Test to verify PVC behavior when full of data with I/O
+"""
+import logging
+import pytest
+import ocs_ci.ocs.exceptions as ex
+
+from ocs_ci.framework.testlib import tier1, E2ETest
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pod import get_used_space_on_mount_point
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    argnames=["interface"],
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL],
+            marks=pytest.mark.polarion_id("OCS-852")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM],
+            marks=[
+                pytest.mark.polarion_id("OCS-853"),
+                pytest.mark.bugzilla("1745344")
+            ]
+        ),
+    ]
+)
+@tier1
+class TestPVCFullWithIORWO(E2ETest):
+    """
+    Test to verify PVC behavior when full with data
+    """
+    pvc_size_gb = 50
+
+    @pytest.fixture()
+    def base_setup(
+        self, request, interface, access_mode, pvc_factory, pod_factory
+    ):
+        """
+        A setup phase for the test
+        """
+        self.interface = interface
+        self.pvc_obj = pvc_factory(
+            interface=self.interface, size=self.pvc_size_gb,
+            access_mode=access_mode
+        )
+        self.pod_obj = pod_factory(interface=self.interface, pvc=self.pvc_obj)
+
+    @pytest.mark.usefixtures(base_setup.__name__)
+    def test_pvc_no_space_left(self, pod_factory):
+        """
+        Writing data to PVC to reach limit
+        """
+        log.info(f"Running FIO to fill PVC size: {self.pvc_size_gb}")
+        self.pod_obj.run_io(
+            'fs', size=self.pvc_size_gb, io_direction='write', runtime=60
```

Comment:
in case the PVC won't be filled in 60 seconds, the test results could be affected

Reply:
done